File: 80-remove-duplicates-from-sorted-array-ii/remove-duplicates-from-sorted-array-ii.java
class Solution {
    public int removeDuplicates(int[] nums) {
        int k=2;

        for(int i=2;i<nums.length;i++){
            if(nums[i]!=nums[k-2]){
                nums[k]=nums[i];
                k++;
            }
        }
        return k;
    }
}

File: 80-remove-duplicates-from-sorted-array-ii/remove-duplicates-from-sorted-array-ii.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        int count=0;
        int n=nums.size();
        unordered_map<int,int> mp;
        for(int i=0;i<n;i++){
            mp[nums[i]]++;
           
            if(mp[nums[i]]<=2){
                cout<<nums[i]<<" :"<<mp[nums[i]]<<endl;
                nums[count]=nums[i];
                count++;
            }
        }
        return count;
    }
};